Repository/AuthenticationService.cs

```diff
@@ -1,57 +1,49 @@
+using GameStore.Dtos;
 using GameStore.Models;
 using GameStore.Shared.Helpers;
 using GameStore.Shared.Services;
-using Blazored.LocalStorage;
+using GameStore.Shared.States;
 using System.IdentityModel.Tokens.Jwt;
 using System.Security.Claims;
 
 namespace GameStore.Repository;
 
-public class AuthenticationService(IHttpService httpService, ILocalStorageService localStorage) : IAuthenticationService
+public class AuthenticationService(IHttpService httpService, LocalStorage localStorage) : IAuthenticationService
```

Comment:
Gw rasa single responsibilty nya Authentication Service cuma buat dapetin LoginResponse, untuk simpen,hapus, baca si JWT token nya bisa di class AuthState

Reply:
action plan: revert to ILocalStorageService